apps/web/lib/surveyLogic/utils.ts

```
@@ -510,25 +511,26 @@ const getLeftOperandValue = (
   selectedLanguage: string
 ) => {
   switch (leftOperand.type) {
-    case "question":
-      const currentQuestion = localSurvey.questions.find((q) => q.id === leftOperand.value);
-      if (!currentQuestion) return undefined;
+    case "element":
+      const elements = getElementsFromBlocks(localSurvey.blocks);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)